src/libostree/ostree-bootloader-zipl.c

```diff
@@ -109,6 +110,14 @@ _ostree_bootloader_zipl_write_config (OstreeBootloader  *bootloader,
   return TRUE;
 }
 
+static gboolean _ostree_secure_execution_is_enabled (GCancellable *cancellable) {
```

Comment:
I'm surprised at this signature (no `GError`). We need to make really sure here that we discern between "Secure Execution is disabled" (OK) and "couldn't read Secure Execution enablement" (hard error).

Is `/sys/firmware/uv/prot_virt_guest` always expected to exist on s390x machines, even if Secure Execution is disabled?

Reply:
no GError, becase `/sys/firmware/uv/prot_virt_guest` exists only on machines with enabled SecureExecution. so if `openat` fails, error isn't propagated to the caller

Reply:
Then ideally, to be strict we should handle `ENOENT` gracefully here but otherwise error out.

Reply:
https://github.com/ostreedev/ostree/pull/2615